balancer/grpclb/statedata/statedata.go

```diff
@@ -0,0 +1,51 @@
+/*
+ *
+ * Copyright 2020 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+// Package statedata declares grpclb types to be set by resolvers wishing to
+// pass information to grpclb via resolver.State Attributes.
+package statedata
+
+import (
+	"google.golang.org/grpc/resolver"
+)
+
+// stateDataAttributesKey is the key to use for storing StateData in Attributes.
+type sdKeyType string
+
+const sdKey = sdKeyType("grpc.grpclb.state_data")
```

Comment:
Done.  Maybe there are too many "states" now, though?

Reply:
What other states? Those are types, not variables?

Reply:
`resolver.State`, `balancer.State`, now a `state` package.  With import renaming this one isn't really a problem in practice, but it could be confusing to have so many things called "state".